### DISCUSSION TITLE
Pinned screenshots are not displayed on the taskbar. Is it possible to bring back the old behavior?

### DISCUSSION THREAD

**Discussion by 87elijah87** [flameshot-org/flameshot#2832] [General]
Hello. Pinned screenshots are not displayed on the taskbar. Is it possible to bring back the old behavior? This option has been removed from recent versions.

![2022-07-26_12-51](https://user-images.githubusercontent.com/21132381/180978013-680aba67-0112-4064-afe5-55f85f24d6ef.png)

**Comment by mmahmoudian** [comment]
I also [noticed this 6 days ago](https://github.com/flameshot-org/flameshot/issues/2768#issuecomment-1190160851) and I think it is connected to #2768. I personally don't clearly remember when this was changed and based on which commit, but I also agree that this should be fixed. I suggest following this case from #2768.
